### ISSUE TITLE
Can you share a way convert model pt to ncnn

### ISSUE THREAD

**Issue by nqthai309** [FeiGeChuanShu/yolov5-seg-ncnn#5]
i used to convert pretrained model yolov5s-seg.pt to ncnn by pnnx tool, but its content of param file is not same content of your param file provided 

this is command convert model: ./pnnx yolov5s-seg.torchscript inputshape=[1,3,640,640]

**Comment by xellDart** [comment]
I have same problem, I try onnx to ncnn and torchscript to pnnx to ncnn

**Comment by superbayes** [comment]
same problem
@FeiGeChuanShu 飞哥救急呀,如何将yolov5s-seg.pt(v7.0) 转换为ncnn?

**Comment by Digital2Slave** [comment]
Refer @FeiGeChuanShu  https://github.com/FeiGeChuanShu/yolov5-seg-ncnn/issues/1#issuecomment-1253376316
1. export the torchscript file by [export.py](https://github.com/ultralytics/yolov5/blob/master/export.py).
2. use [pnnx](https://github.com/Tencent/ncnn/tree/master/tools/pnnx) tool to convert the torchscript to ncnn model.

How to modify the following commands to generate the correct ncnn model files.

```
$ python export.py --weights yolov5s-seg.pt --include torchscript
$ ./pnnx yolov5s-seg.torchscript inputshape=[1,3,640,640]
```

Note that,  **yolov5s-seg.pt** is downloaded from **yolov5-7.0**. 

https://github.com/ultralytics/yolov5/releases/download/v7.0/yolov5s-seg.pt

**Comment by nqthai309** [comment]
> Refer @FeiGeChuanShu [#1 (comment)](https://github.com/FeiGeChuanShu/yolov5-seg-ncnn/issues/1#issuecomment-1253376316)
> 
> 1. export the torchscript file by [export.py](https://github.com/ultralytics/yolov5/blob/master/export.py).
> 2. use [pnnx](https://github.com/Tencent/ncnn/tree/master/tools/pnnx) tool to convert the torchscript to ncnn model.
> 
> How to modify the following commands to generate the correct ncnn model files.
> 
> ```
> $ python export.py --weights yolov5s-seg.pt --include torchscript
> $ ./pnnx yolov5s-seg.torchscript inputshape=[1,3,640,640]
> ```
> 
> Note that, **yolov5s-seg.pt** is downloaded from **yolov5-7.0**.
> 
> https://github.com/ultralytics/yolov5/releases/download/v7.0/yolov5s-seg.pt

output and input of converted model seem different with pre-converted model provided by FeiGeChuanShu.

**Comment by Digital2Slave** [comment]
@nqthai309 yes. i have the same issue. So i write "How to modify the following commands to generate the correct ncnn model files."
 No response from @FeiGeChuanShu. Maybe there is something he modified but do not want to show out, just post [export.py](https://github.com/ultralytics/yolov5/blob/master/export.py) and [pnnx](https://github.com/Tencent/ncnn/tree/master/tools/pnnx) .

**Comment by Digital2Slave** [comment]
@nqthai309 @superbayes @xellDart It seems `block issues` of pnnx .  I have put an issue https://github.com/Tencent/ncnn/issues/4488 

![image](https://user-images.githubusercontent.com/7224107/215308232-82c6e47a-dcbd-49a0-bd93-fad2523da4a4.png)


https://github.com/Tencent/ncnn/issues/4489#issue-1561097873

**Comment by nqthai309** [comment]
@Digital2Slave Thanks bro <3

**Comment by MjiS** [comment]
@Digital2Slave  Any update regarding this issue? I am also facing the same issue, I tried ONNX to NCNN approach also and torchscript > PNNX > NCNN but the output param file is different from provided by @FeiGeChuanShu . 
It seems he used a different approach :/

**Comment by Digital2Slave** [comment]
@MjiS I change to use yolov8 segment model. You can refer wiki https://github.com/Digital2Slave/ncnn-android-yolov8-seg/wiki/Conver-yolov8-model-to-ncnn-model.

By the way, because the yolov5 segment model may different  between versions. The convert NCNN model file maybe also different from @FeiGeChuanShu .

**Comment by MjiS** [comment]
@Digital2Slave Thank you for your guidance but I want to run segmentation in real time like yolov5-seg. There is no app for Yolov8s-seg model or do I need to make specific changes to it to run the segmentation model?

**Comment by Digital2Slave** [comment]
> @Digital2Slave Thank you for your guidance but I want to run segmentation in real time like yolov5-seg. There is no app for Yolov8s-seg model or do I need to make specific changes to it to run the segmentation model?

Refer https://github.com/Digital2Slave/ncnn-android-yolov8-seg
